conf/CMakeLists.txt

```diff
@@ -20,4 +20,4 @@ configure_file(
 
 # Install the yaml configuration files in an unversioned location.
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${IGN_DESIGNATION}${PROJECT_VERSION_MAJOR}.yaml
-  DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATAROOTDIR}/ignition/)
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATAROOTDIR}/gz/)
```

Comment:
This breaks the `ign msg` CLI. I believe we haven't been migrating the install paths for these yaml files yet and `gz-tools` is still looking at `install/share/ignition`

Reply:
https://github.com/ignitionrobotics/ign-msgs/pull/249/commits/69bdd6a8ba7c68cff6783ec93233102aaa49fc1e